src/DependencyInjection/Configuration.php

```diff
@@ -11,9 +11,15 @@ final class Configuration implements ConfigurationInterface
     /** {@inheritdoc} */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $builder = new TreeBuilder();
-
-        $root = $builder->root('lamoda_enum');
+        $version = \PackageVersions\Versions::getVersion('symfony/dependency-injection');
```

Comment:
This thing is broken when it comes to transitive dependencies (https://github.com/Ocramius/PackageVersions/issues/116) . The common approach is to rely on `method_exists` to determine current API version

https://github.com/doctrine/DoctrineBundle/commit/e711bed5a82437262aeade06373fcd5c81a2fd94